orm/encoding/ormvalue/string.go

```diff
@@ -0,0 +1,88 @@
+package ormvalue
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"strings"
+
+	"google.golang.org/protobuf/reflect/protoreflect"
+)
+
+// StringCodec encodes strings as raw bytes.
+type StringCodec struct{}
+
+func (s StringCodec) FixedSize() int {
+	return -1
+}
+
+func (s StringCodec) Size(value protoreflect.Value) (int, error) {
+	return len(value.String()), nil
```

Comment:
Usage of .String() might be dangerous as it's possible to feed a protoreflect int64 values and string will return the formatted version of the int64

Reply:
good catch 👍